index.html

```diff
@@ -1947,10 +1947,10 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
         <p data-lang="zh-hans" class="checkme">若无特别示，本文使用的“阿拉伯数字”皆指西方语言或欧洲形式的印度-阿拉伯数字。</p>
         <p data-lang="zh-hant">若無特別提示，本文使用之「阿拉伯數字」皆指西方語言或歐洲形式的印度－阿拉伯數字。</p>
       </div>
-      <p data-lang="en">Formerly, fullwidth ASCII characters were often used, either to make the presentation look orderly, or simply due to the poorly developed computer technologies available for text layout. Nowadays, typesetting engines allow for proportional or monospace fonts, as required, rather than forcing the user to resort to the old  fullwidth blocks of Latin letters and European numerals.</p>
+      <p data-lang="en">Formerly, full-width ASCII characters were often used, either to make the presentation look orderly, or simply due to the poorly developed computer technologies available for text layout. Nowadays, typesetting engines allow for proportional or monospace fonts, as required, rather than forcing the user to resort to the old  full-width blocks of Latin letters and European numerals.</p>
```

Comment:
See above.

Reply:
I'll revert them all back to the versions with dash